openqa-label-known-issues

```diff
@@ -142,6 +142,10 @@ investigate_issue() {
     # see https://progress.opensuse.org/issues/69178
     echo "$reason" >> "$out"
     if [[ "$curl_output" != "200" ]] && [[ "$curl_output" != "301" ]]; then
+        # Assume autoinst-log.txt still not found but process job anyway due to reason var
+        if [[ -n $reason ]] && [[ "$curl_output" = "404" ]] && [[ $reason != null ]]; then
```

Comment:
For the sake of consistency:
```suggestion
        if [[ -n $reason ]] && [[ $curl_output = 404 ]] && [[ $reason != null ]]; then
```

Reply:
e87edaa done

Reply:
I don't see that change applied. There's still `"` in the code 🤔

Reply:
I can see it https://github.com/os-autoinst/scripts/pull/335/files#diff-4b82b9655e797531626dd3da7dce9a5f1e1bcade526ba3ad0262a1a4569a3570R141

Reply:
There are still quotes around the `404`. I guess it doesn't really matter.